posixlib/src/main/resources/scala-native/signal.h

```diff
@@ -0,0 +1,6 @@
+#ifndef SIGNAL_H
```

Comment:
Looks like this header is not needed right now, it can probably be removed.

Reply:
Originally added for `aio` so removed for now.